test/Test/Database/LSMTree/Normal/StateMachine.hs

```diff
@@ -1082,9 +1083,10 @@ arbitraryActionWithVars _ findVars _st = QC.frequency $ concat [
     genDeletes :: Gen (V.Vector k)
     genDeletes = QC.arbitrary
 
-    -- TODO: generate @Just blob@ once blob references are implemented
     genBlob :: Gen (Maybe blob)
-    genBlob = Nothing <$ QC.arbitrary @blob
+    genBlob = QC.oneof [ pure Nothing
+                       , Just <$> QC.arbitrary @blob
+                       ]
```

Comment:
Isn't this just `QC.arbitrary @(Maybe blob)`?

Reply:
Um yes. :grin: